app-backend/api/src/main/scala/categories/Routes.scala

```diff
@@ -50,7 +50,7 @@ trait ToolCategoryRoutes
       complete {
         ToolCategoryDao.query
           .filter(combinedParams)
-          .page(page)
+          .page(page, fr"")
```

Comment:
man, it really rubs me the wrong way that we have to do this :man_facepalming:

Reply:
Me too! Also @alkamin. No one is happy that `assembly` throws our methods in the garbage :man_shrugging: